digital-signage-app-v2/app/components/content-area.js

```diff
@@ -0,0 +1,93 @@
+import Ember from 'ember';
```

Comment:
This exists because it is easier to control data that is in a component template rather than in a route template.